### PR TITLE
Adapt EG to arm template

### PR DESCRIPTION
**Pull request by rakshith91**
Fixes https://github.com/Azure/azure-sdk-for-python/issues/15967

### CODE REVIEW

File: sdk/eventgrid/azure-eventgrid/samples/publish_samples/publish_cloud_events_to_domain_topic_sample.py [hunk 1]
```diff
@@ -46,7 +44,7 @@ def publish_event():
             sample_members = sample(services, k=randint(1, 4))      # select random subset of team members
             event = CloudEvent(
                     type="Azure.Sdk.Demo",
-                    source=domain_name,
+                    source='/azsdk/members',
```
Comment (yunhaoling) [inline sdk/eventgrid/azure-eventgrid/samples/publish_samples/publish_cloud_events_to_domain_topic_sample.py:47]: nit: probably worth a better const string than `'/azsdk/members'`? like `'/demo/domain_name'`

**Review by yunhaoling — APPROVED** [review]

File: sdk/eventgrid/azure-eventgrid/samples/publish_samples/publish_cloud_events_to_domain_topic_sample.py (hunk 1 above)
Comment (rakshith91) [reply sdk/eventgrid/azure-eventgrid/samples/publish_samples/publish_cloud_events_to_domain_topic_sample.py:47]:
```suggestion
                    source='/demo/domain_name',
```

**Pull request merged**